### DISCUSSION TITLE
Dialog, How to Scroll to top if the dialog re open?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuetifyjs/vuetify#13453] [Q&A]
So I was having trouble in my NuxtJs app, so I have this dialog, which is scrollable who are being accessed by a different button. When I open the dialog, it doesn't scroll to the top instead, it is in the same spot.

**Comment by jacekkarczmarczyk** [comment]
You need to do it manually after opening the dialog

https://codepen.io/jkarczm/pen/PoWyWOm?editors=1010

**Reply by JenuelDev** [reply]
thanks, I will try this later, and I will come back. :)
